BranchChangeLog

```diff
@@ -1,3 +1,31 @@
+===============================================================
+Tag Creator: klindsay
+Developers:  klindsay
+Tag Date:    21 Jun 2020
+Tag Name:    pop2/trunk_tags/cesm_pop_2_1_20190306
```

Comment:
You have `Tag Name` listed twice, and I think the `trunk_tag` name is incorrect

Reply:
extra `Tag Name` line removed

@mnlevy1981 , I haven't actually made the tag, assuming that this step is performed after the PR is merged.
Is that correct?

Reply:
@klindsay28 That's right -- I'll merge the branch and then create the tag. I can update the Tag Date when I do the merge (`git merge --no-commit --no-ff` will let me modify files the same way we modify files if there is a conflict in the merge). I guess I could've cleaned up the tag name as well...